ern-api-impl-gen/src/generators/ApiImplGen.js

```diff
@@ -31,11 +36,15 @@ export default class ApiImplGen {
 
     await this.downloadApiAndDependencies(apiDependency, paths.pluginsDownloadDirectory, reactNativeVersion)
 
+    const schemaJson = path.join(paths.pluginsDownloadDirectory, `node_modules`, apiDependency.scopedName, `schema.json`)
```

Comment:
`package.json` and `schema.json` can go as a Constant as done in `ern-api-gen` module, if it is used at 1 or more places, we can import as a contant.

Reply:
This is done dynamically, this dependency will only be downloaded at this point.